### ISSUE TITLE
AR plot - mixed up axis labels

### ISSUE THREAD

**Issue by daniel-d-koenig** [votchallenge/vot-toolkit-python#19]
Hi,
I think there is a little typo of the axis labels in AR plot. The following line defines the 'Robustness' and 'Accuracy' labels:
https://github.com/votchallenge/vot-toolkit-python/blob/52b5559d74a16518c5970c2f56eacc06cc0a715a/vot/analysis/ar.py#L55

In the following table I have Accuracy = 0.37 and Robustness = 0.01 for evaluation metric values, but in den plot the position of tracker in the AR plot is at around 0.4 for Robustness and 0.1 for Accuracy. I think either the labels or the evaluation results should by changed.
![image](https://user-images.githubusercontent.com/65339272/86747210-e2450900-c03b-11ea-9f71-181f752446ad.png)
![image](https://user-images.githubusercontent.com/65339272/86747058-c9d4ee80-c03b-11ea-88eb-14fdc9b24257.png)
![image](https://user-images.githubusercontent.com/65339272/86743377-1f5bcc00-c039-11ea-8f7b-99f801cd4b50.png)

Thanks in advance.

Best regards,
Daniel

**Comment by lukacu** [comment]
Indeed, I think we have fixed this in dev branch, but it got stuck there because I am busy with other projects at the moment.

**Comment by daniel-d-koenig** [comment]
I installed the current dev branch and generated the graphs, but unfortunately the data is still interchanged. Data on the x axis should be on y axis and vice versa. 

Maybe in the following line the indices of data should be changed 
https://github.com/votchallenge/vot-toolkit-python/blob/458003f27d2f759fd6ada6f237323a1a01e75cd4/vot/document/__init__.py#L76
to
`handle = self._axes.scatter(data[1], data[0], **style.point_style())`
this will lead that 'Accuracy' values are on y axis and 'Robustness' values on x axis.

Another possibility could be to change following function:
https://github.com/votchallenge/vot-toolkit-python/blob/458003f27d2f759fd6ada6f237323a1a01e75cd4/vot/analysis/ar.py#L53
